Makefile

```diff
@@ -1,5 +1,9 @@
 all: build
 
+build_date := $(shell date +%s)
+build_commit := $(shell git rev-parse --short HEAD)
```

Comment:
nit: would be nice use the tag if available. See https://github.com/istio/istio/blob/974351942aa08ec2d9f2f6f350b9733b0f6130e4/bin/get_workspace_status.sh#L17-L23 and https://github.com/istio/istio/blob/974351942aa08ec2d9f2f6f350b9733b0f6130e4/bin/get_workspace_status.sh#L32-L41.